tensorflow/compiler/mlir/tosa/transforms/legalize_common.cc

```diff
@@ -2797,5 +2980,563 @@ llvm::Optional<Value> convertTFConv2DCommon(
       .getResult();
 }
 
+// Lowers Gather operators to a sequence of TOSA ops.
+llvm::Optional<Value> convertGatherOp(PatternRewriter& rewriter, Operation* op,
+                                      Value result_value, Value params_value,
+                                      Value indices_value, int32_t batch_dims,
+                                      int32_t axis)
+
+{
```

Comment:
nit: remove newline and put opening brace inline with function defn.

Reply:
Thanks. Will update in next round.